README.md

```diff
@@ -22,11 +22,11 @@ Designed for ease of use, future scalability, and unparalleled performance, this
 
 All the challenging build settings are automatically handled by this package.
 
-| Platform \ Target | Android | iOS/iPadOS | Linux | macOS | Windows | Web |
-| ----------------- | ------- | ---------- | ----- | ----- | ------- | --- |
-| Linux             | ✅      | -          | ✅    | -     | -       | ✅  |
-| macOS             | ✅      | ✅         | -     | ✅    | -       | ✅  |
-| Windows           | ✅      | -          | -     | -     | ✅      | ✅  |
+| Dev     | Android | iOS | Linux | macOS | Windows | Web |
```

Comment:
Hi, is  "Platform \ Target" changed to "Dev" maybe a mistake?

Reply:
This means 'Dev platform', which points to linux/window/macos below. I made this change because I thought the column names should represent things 'below', not 'sideways'.

I will change the name to 'Dev OS' just to make it more clear. I wanted it to be shorter, too :)